source/workers/LocalWorker.cpp

```diff
@@ -4105,6 +4315,13 @@ void LocalWorker::s3ModeUploadObjectSinglePart(std::string bucketName, std::stri
 		.WithKey(objectName)
 		.WithContentLength(blockSize);
 
+    if (progArgs->getDoS3ObjectTagInline())
+    {
+        request.SetTagging(
+                "elbencho_random_tag=" + StringTk::generateRandomS3TagValue(objectName, 30)
```

Comment:
let's have a `#define` for the `30`, similar to the corresponding defines for bucket tags